.buildbot.sh

```diff
@@ -24,18 +24,19 @@ tar -xf llvm-project-13.0.0.src.tar.xz
 cd llvm-project-13.0.0.src
 mkdir -p build
 cd build
-cmake -DCMAKE_BUILD_PARALLEL_LEVEL=24 -DLLVM_ENABLE_PROJECTS="clang;llvm;lld;clang-tools-extra" -DLLVM_LINK_LLVM_DYLIB=ON -G "Unix Makefiles" ../llvm/.
-cmake --build .
+cmake -DLLVM_ENABLE_PROJECTS="clang;llvm;lld;clang-tools-extra" -DLLVM_LINK_LLVM_DYLIB=ON -G "Unix Makefiles" ../llvm/.
```

Comment:
Is the `.` at the end of the line necessary? [I think it's superfluous but I'm not 100% sure.]

Reply:
I have just followed the documentation for building `llvm`.

Reply:
Link?

Reply:
Yes, the "." at the end of the line is not doing anything useful AFAICS.

Reply:
Fix in [3a25324](https://github.com/capablevms/llvm-function-split/pull/26/commits/3a2532425112c699606b8a3fe860d7e56ca3dfea).